### ISSUE TITLE
Lock npm/ bower version

### ISSUE THREAD

**Issue by amitaibu** [Gizra/generator-hedley#82]
Find the installed npm versions -- http://stackoverflow.com/a/16704412/750039

For bower `bower list` seems to do the work as-well, but slightly harder to read

**Comment by ordavidil** [comment]
npm seems to be [locked already](https://github.com/Gizra/generator-hedley/blob/master/app/templates/client/package.json#L6-L31)

**Comment by ordavidil** [comment]
Bower seems to be [locked](https://github.com/Gizra/generator-hedley/blob/master/app/templates/client/bower.json) as well besides the angular resolution, I'll open a PR for that.
The minor version is not locked for some of the packages, should we lock that too? As I know, minor versions don't make any big change.

**Comment by amitaibu** [comment]
No, it's not locked.  It should have the explicit version. Minor version
changes might still break stuff - as there is no api gurantee like in elm.
